{{cookiecutter.project_name}}/.github/workflows/test.yaml

```diff
@@ -18,27 +18,42 @@ defaults:
     shell: bash -euo pipefail {0}
 
 jobs:
+  get-environments:
```

Comment:
How about adding some comments on what this is doing and why we think this is the preferred way?

Reply:
Sounds good!